### PR TITLE
Symflower.SymflowerCLI version 20956

### PR DESCRIPTION
**Pull request by bauersimon**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/50135)

### CODE REVIEW

File: manifests/s/Symflower/SymflowerCLI/20890/Symflower.SymflowerCLI.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Symflower.SymflowerCLI
+PackageVersion: 20890
+PackageLocale: en-US
+Publisher: Symflower GmbH
+PackageName: Symflower CLI
+License: Proprietary
+LicenseURL: https://symflower.com/en/license/
```
Comment (ItzLevvie) [inline manifests/s/Symflower/SymflowerCLI/20890/Symflower.SymflowerCLI.locale.en-US.yaml:10]:
```suggestion
LicenseUrl: https://symflower.com/en/license/
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Symflower/SymflowerCLI/20890/Symflower.SymflowerCLI.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Symflower.SymflowerCLI
+PackageVersion: 20890
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
```
Comment (ItzLevvie) [inline manifests/s/Symflower/SymflowerCLI/20890/Symflower.SymflowerCLI.installer.yaml:8]:
```suggestion
  Architecture: x64
```
Installer is `x86_64` which is meant to be run on x64 devices.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/s/Symflower/SymflowerCLI/20890/Symflower.SymflowerCLI.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Symflower.SymflowerCLI
+PackageVersion: 20890
```
[the remark this answers is not in the document]
Comment (bauersimon) [reply manifests/s/Symflower/SymflowerCLI/20890/Symflower.SymflowerCLI.locale.en-US.yaml]: I also renamed the folder to 1.0.20890 then.

File: manifests/s/Symflower/SymflowerCLI/1.0.20917/Symflower.SymflowerCLI.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: Symflower.SymflowerCLI
+PackageVersion: 1.0.20917
+Installers:
+  - InstallerLocale: en-US
+    Architecture: x64
+    InstallerType: wix
+    InstallerUrl: https://download.symflower.com/local/latest/symflower-windows-x86_64.msi
+    InstallerSha256: 7B61FE2B245B70CA950393890FCDBE8586B40CB25EC9260E5304554D6714730E
+    ProductCode: "{8E68DE64-6D60-42CE-88C4-290CECBAFA6F}"
```
Comment (Trenly) [inline manifests/s/Symflower/SymflowerCLI/1.0.20917/Symflower.SymflowerCLI.installer.yaml:12]:
```suggestion
    ProductCode: '{B41C1F22-E777-4C3D-ACDC-944B9CA8482E}'
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**